src/coreComponents/mesh/generators/CellBlockManagerABC.hpp

```diff
@@ -99,12 +101,26 @@ class CellBlockManagerABC : public dataRepository::Group
    */
   virtual Group & getFaceBlocks() = 0;
 
+  /**
+   * @brief Returns LineBlockABC corresponding to the given identifier
+   * @param name the name of the required LineBlockABC
+   * @return The LineBlockABC associated with the given name
+   *
+   */
+  virtual LineBlockABC const & getLineBlock( string name ) const = 0;
+
   /**
    * @brief Returns a group containing the cell blocks as CellBlockABC instances
    * @return Const reference to the Group instance.
    */
   virtual const Group & getCellBlocks() const = 0;
 
+  /**
+   * @brief Returns a group containing the face blocks as FaceBlockABC instances
+   * @return Const reference to the Group instance.
+   */
+  virtual const Group & getFaceBlocks() const = 0;
```

Comment:
Do you now if we still need the non const versions of the `getCellBlocks`, `getFaceBlocks`?

Reply:
/appli_RD/LACOSTE/GEOSX/GEOSX/src/coreComponents/mesh/ElementRegionManager.cpp:130:    elemRegion.generateMesh( cellBlockManager.getFaceBlocks() );
/appli_RD/LACOSTE/GEOSX/GEOSX/src/coreComponents/mesh/ElementRegionManager.cpp:126:    elemRegion.generateMesh( cellBlockManager.getCellBlocks() );                                  |